src/modules/visits/services/visits.service.ts

```diff
@@ -205,6 +215,12 @@ export class VisitsService {
 			};
 		}
 
+		if (started) {
+			where.start_date = {
+				_lte: new Date().toISOString(),
+			};
+		}
```

Comment:
This might be out of scope of the task, but the reverse filter doesn't seem to work: If you set started to false i would expect it to only show the visits where startAt < now()

![image](https://user-images.githubusercontent.com/1710840/157424692-16b6fc38-b5e5-416a-868e-7fe326528d3c.png)

```suggestion
		if (started) {
			where.start_date = {
				_lte: new Date().toISOString(),
			};
		} else if(!_.isUndefined(started)) {
			where.start_date = {
				_gt: new Date().toISOString(),
			};
		}
```